### DISCUSSION TITLE
eval export HOMEBREW_PREFIX=/opt/homebrew - HELP PLEASE

### DISCUSSION THREAD

**Discussion by bebz00** [Homebrew/discussions#3099] [Getting started]
Hi, I installed HomeBrew on my M1 Mac mini, and now every time I open the terminal, I am greeted with this message, eval export HOMEBREW_PREFIX=/opt/homebrew, 

![Screenshot 2022-03-20 at 14 58 11](https://user-images.githubusercontent.com/33040079/159168744-07b67888-f702-4758-aae5-b8f4c3c1e103.png)

please can someone help, no commands work, when Homebrew was installed I had recently installed Anaconda and that is the only opt file and nothing to suggest homebrew.

![Screenshot 2022-03-20 at 14 58 42](https://user-images.githubusercontent.com/33040079/159168753-05c6e74b-712f-4455-8adc-4e0fa902b621.png)

please please please, can some one explain what I have done wrong and how to repair this.

Many thanks in advance.
Bob

**Comment by carlocab** [comment]
What's the output of
```
cat ~/.zprofile
```
?

**Comment by bebz00** [comment]
Hi Carlo
There is no output..
I get no response from any commands.


On 20 Mar 2022, at 15:16, Carlo Cabrera ***@***.******@***.***>> wrote:



[cid:3A597C45-2B52-4F4E-92BD-D114662BB85E]What's the output of

cat ~/.zprofile


?

—
Reply to this email directly, view it on GitHub<https://github.com/Homebrew/discussions/discussions/3099#discussioncomment-2397643>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AH4CNTYLZXAVDW57SPWYXTLVA46LZANCNFSM5RFT62QQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by carlocab** [reply]
What about `echo $SHELL` and `cat ~/.zshrc`?

**Reply by carlocab** [reply]
Oh, wait, that probably won't work either. It seems you've broken your shell startup script somehow. It's not really clear how exactly, though.

**Comment by bebz00** [comment]
Is there a way that I can repair the shell script, or be able to uninstall Homebrew and anaconda.

Regards
Bob

On 20 Mar 2022, at 15:49, Carlo Cabrera ***@***.******@***.***>> wrote:



Oh, wait, that probably won't work either. It seems you've broken your shell startup script somehow. It's not really clear how exactly, though.

—
Reply to this email directly, view it on GitHub<https://github.com/Homebrew/discussions/discussions/3099#discussioncomment-2397765>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AH4CNT6RELM2CBGHN7PNUKTVA5CJTANCNFSM5RFT62QQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by mistydemeo** [comment]
If you open the Terminal preferences, you should be able to change the shell to use when creating new sessions. Can you change the "shells open with" setting to "command", and enter `/bin/bash` like this? That should open an alternative shell for you, skipping the bad configuration and allowing you to check the settings that @carlocab asked for.

<img width="779" alt="image" src="https://user-images.githubusercontent.com/780485/159346144-fcceea8b-235e-4944-a3a8-4b098a182751.png">

**Reply by Burritbro** [reply]
I did that and uninstalled homebrew. My regular terminal works but the terminal in vs code and pycharm both have the line "eval export HOMEBREW_PREFIX=/opt/homebrew" and no inputs do anything.  I reinstalled both ide's twice after uninstalling homebrew and the issue is not fixed. Any thoughts?
<img width="1245" alt="Screen Shot 2022-12-28 at 2 57 18 PM" src="https://user-images.githubusercontent.com/108247464/209865684-bac97353-1362-42a5-aa67-19d8228b4170.png">
